### ISSUE TITLE
Documentation -  CI/CD technology

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#23]
## Acceptance criteria
* Section in Wiki to describe FE technology decision making
* Small listing of discussed technologies

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
